india_compliance/gst_india/client_scripts/e_invoice_actions.js

```diff
@@ -160,10 +160,31 @@ function show_cancel_e_invoice_dialog(frm, callback) {
 function is_e_invoice_applicable(frm) {
     return (
         india_compliance.is_e_invoice_enabled() &&
-        frm.doc.docstatus == 1 &&
+        // frm.doc.docstatus == 1 &&
```

Comment:
Is this intentional?

Reply:
It was for testing. I forgot to revert this change